AirCasting/Onboarding/NearAirDescription.swift

```diff
@@ -6,15 +6,15 @@ import AirCastingStyling
 
 struct NearAirDescription: View {
     var completion: () -> Void
+    
     var body: some View {
-        VStack(alignment: .leading, spacing: 25) {
+        VStack(alignment: .leading) {
```

Comment:
Could you horizontally center the button on this screen as well? It's a bit to the left

Reply:
This button is actually a navigation link, and for some reason, it doesn't take up the whole space, so there is a bigger gap between the button and the edge of the screen on the right thank on the left. Maybe you'll be able to see it when you change the alignment to `.center` and see it move a bit.
<img width="1054" alt="Screenshot 2022-07-05 at 10 39 50" src="https://user-images.githubusercontent.com/15946812/177287321-9895ebcb-fc78-44ba-94dd-a13cd56312df.png">
<img width="1050" alt="Screenshot 2022-07-05 at 10 40 14" src="https://user-images.githubusercontent.com/15946812/177287398-a1d6934d-5677-4d13-afad-bac68360489e.png">

Reply:
But probably the alignment of the VStack shouldn't be changed and just the button should be centered (or bigger)